.github/workflows/lint.yaml

```diff
@@ -0,0 +1,88 @@
+name: "Lint"
+
+on: [push, pull_request]
+
+env:
+    fail-fast: true
+
+jobs:
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: PHP-CS-Fixer
+              uses: docker://oskarstark/php-cs-fixer-ga
+              with:
+                  args: --diff --dry-run
+
+    linters:
+        name: Linters
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                allow-failures: [false]
+                operating-system: ['ubuntu-latest']
+                php-version: ['7.4']
```

Comment:
As there is no valid reason to run our linters on several PHP versions (the result does not depend on the PHP version), I would remove the usage of a single-value matrix here and put this `7.4` directly in the config of `shivammathur/setup-php`

Reply:
I know this is not exactly need, but it is also used in the composer cache key so it is some kind of variable which is very well to read via: `${{ matrix.php-version }}`

I am for keeping the matrix for `php-version`

Reply:
Both of you have good arguments ... but let's keep it in the matrix for now.